client/src/components/game/lab2/Instructions/thirdInstructions.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Button from '../components/header/buttons/button';
+import './secondaryInstructions.css';
+
+/*
+Page containing the instructions for the second popup the user sees
+this instructional popup covers color contrast
+*/
+const ThirdInstructions  = ({closePage, activatePopup, toWhiteBackground,
+  background, selectOption}) => {
+
+  let alreadyCalled = false;
+  if (!alreadyCalled) {
+    selectOption('Protanopia');
+  }
+
+  const changeColors = () => {
+    activatePopup();
+  }
+
+  if (background !== 'white') {
+    toWhiteBackground();
+  }
+
+  return (
+    <div className="instructionsContainer">
+      <p className='secondInstructionTitle'>Instructions -- Part Three</p>
```

Comment:
Since the first page of instructions and second page of instructions don't say "Instructions -- Part One" and "Instructions -- Part Two", I think saying "Part Three" here may not be necessary. Or maybe add "Part One" and "Part Two" to the first two pages of instructions for consistency?